snapcraft/reference/debapt-snap-comparison.md

```diff
@@ -0,0 +1,42 @@
+# APT/deb - snap Packaging Comparison
+
+These tables provide a comparison between the APT/deb and snap package management approaches, highlighting key differences and considerations for developers, maintainers, and end-users.
+
+| **Feature** | **APT/deb Package Management** | **snap Package Management** |
+|---|---|---|
+| **Platform Support** | Debian-based systems | Cross-distribution (Ubuntu, Fedora, etc.) |
+| **Package Format** | .deb files | .snap files |
+| **Dependency Handling** | Requires manual dependency management | Bundles dependencies within the package |
+| **Security** | Standard Linux permissions | Sandboxing, AppArmor, Seccomp |
+| **Updates** | Manual updates via package managers (APT) | Automatic updates |
+| **Rollback** | Requires manual reinstallation of previous versions | Built-in rollback to previous versions via `snap revert` |
+| **Installation** | `apt install <deb name>` | `snap install <snap name>` |
+| **Package Management Tools** | **APT** (`apt`, `dpkg`, `aptitude`) | **SNAP** (`snap`, `snapd`) |
+| **Isolation** | No isolation by default | Sandboxed environment |
+| **Channels** | Not applicable | Stable, candidate, beta, edge |
+| **User Data Storage** | Managed by the package itself or user configurations | Separate data storage within confined space |
+| **Package Distribution** | Centralized repositories (e.g., Debian archives) | Centralized via Snap Store |
+| **Development Considerations** | Follow Debian packaging guidelines, ensure compatibility | Self-contained, fewer dependency issues |
+| **End-User Experience** | Familiar to Linux users, requires manual handling | Simplifies installation and updates, user-friendly |
+
+## **Package Builder/Maintainer Considerations**
+  
+| **Feature** | **APT/deb Package Management** | **snap Package Management** |
+|---|---|---|
+| **Build Tools** | `dpkg-deb`, `debuild`, `lintian` | `snapcraft` |
+| **Metadata** | Control files, postinst, prerm scripts | `snapcraft.yaml`, confinement settings |
```

Comment:
Technically the metadata is the `snap.yaml`, not the `snapcraft.yaml` (though the former *is* generated by the latter, snapd doesn't examine the latter and it may not even be included in the final snap package).

Additionally, I wouldn't call `postinst`, `prerm` et al. 'metadata' -- they feel more like extensions of the package itself; the analogue would be snap hooks (`install`, `remove`, etc.); probably more fitting in the `Feature` table.

Reply:
Sorry, hadn't noticed the reply! I think that's sufficient, but admittedly I am unfamiliar with debs.